bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaAccountHandler.java

```diff
@@ -269,33 +261,11 @@ private ThingStatusInfo authenticate() {
         }
 
         if (hasExpired) {
-            String username = (String) getConfig().get(CONFIG_USERNAME);
-            String password = (String) getConfig().get(CONFIG_PASSWORD);
             String refreshToken = (String) getConfig().get(CONFIG_REFRESHTOKEN);
 
             if (refreshToken == null || refreshToken.isEmpty()) {
-                if (username != null && !username.isEmpty() && password != null && !password.isEmpty()) {
-                    try {
-                        refreshToken = ssoHandler.authenticate(username, password);
-                    } catch (Exception e) {
-                        logger.error("An exception occurred while obtaining refresh token with username/password: '{}'",
-                                e.getMessage());
-                    }
-
-                    if (refreshToken != null) {
-                        // store refresh token from SSO endpoint in config, clear the password
-                        Configuration cfg = editConfiguration();
-                        cfg.put(TeslaBindingConstants.CONFIG_REFRESHTOKEN, refreshToken);
-                        cfg.remove(TeslaBindingConstants.CONFIG_PASSWORD);
-                        updateConfiguration(cfg);
-                    } else {
-                        return new ThingStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                                "Failed to obtain refresh token with username/password.");
-                    }
-                } else {
-                    return new ThingStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                            "Neither a refresh token nor credentials are provided.");
-                }
+                return new ThingStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "No refresh token is provided.");
```

Comment:
Consider adding this string to the i18n file for being localizable.

Reply:
This isn't done for thing status updates throughout the whole binding yet, so I'd not want to just do it for this single occasion.